flight/mcu/lib/teensyShared/BarometerSensor.h

```diff
@@ -0,0 +1,17 @@
+#ifndef BAROMETERSENSOR_H
+#define BAROMETERSENSOR_H
+
+#include <MS5611.h>
+
+struct BarometerSensor {
+    MS5611* MS;
+    void readReadings();
+    float getPressure();
+    float getTemperature();
+    float getAltitude();
+    float pressure;
+    float temperature;
+    float altitude;
```

Comment:
Can you make the internals private so that only the get... and read readings functions are public